landing/src/layouts/MainPage.astro

```diff
@@ -109,4 +109,4 @@ const shepherdIncludeCode = `
     </div>
   </main>
   <Footer />
-</Base>
+</Base>
```

Comment:
had to remove the changes here b/c I changed this file to use more as a layout for more pages. The script is in the demo.astro file now, but worse case we capture those in another PR